framework/branch-key-store.md

```diff
@@ -510,6 +649,16 @@ with a request constructed as follows:
 - `DestinationKeyId` MUST be [compatible with](#aws-key-arn-compatibility) the configured KMS Key in the [AWS KMS Configuration](#aws-kms-configuration) for this keystore.
 - `DestinationEncryptionContext` MUST be the [encryption context](#encryption-context) constructed above
 
+#### Authenticating a Branch Keystore item for item with `hierarchy-version` v2
+
+The operation MUST use the configured `KMS SDK Client` to authenticate the value of the keystore item.
+
+This operation MUST call AWS DDB `GetItem`.
+Every attribute except `enc` on the AWS DDB response item MUST be converted to a set of key value pairs (a Map)
+which is the [branch key context](#branch-key-context). The [encryption context](#encryption-context) is built from branch key context by extracting keys with prefix `aws-crypto-ec:` and then dropping the prefix.
+
+The operation MUST follow [AWS KMS Branch Key Decryption](#aws-kms-branch-key-decryption) to decrypt and authentication the branch key context.
```

Comment:
Will this work? "Every key in the constructed [encryption context](https://github.com/awslabs/aws-encryption-sdk-specification/blob/aad0ee3b47e9d3d01a625df8a7db89d40957eac3/framework/branch-key-store.md#encryption-context) except tableName MUST exist as a string attribute in the AWS DDB response item."?